nixos/modules/services/hardware/monado.nix

```diff
@@ -15,6 +15,10 @@ in
 
     package = mkPackageOption pkgs "monado" { };
 
+    layersPackage = mkPackageOption pkgs "monado-vulkan-layers" {};
+
+    hardwarePackage = mkPackageOption pkgs "xr-hardware" {};
```

Comment:
Should we allow users to set `null` to disable either of these?

Reply:
I don't really think there is much of a point to this. If someone really wants to "debloat" their systems, they could set these to an empty derivation.